dags/utils/acoustic/acoustic_client.py

```diff
@@ -152,7 +152,8 @@ def generate_report(
 
         supported_report_types = ("raw_recipient_export", "contact_export",)
         if report_type not in supported_report_types:
-            pass  # TODO: raise with options
```

Comment:
You could omit "pass" here with the same result?

Reply:
this line is being removed with this PR